Ecosystem Partners/Marketplace Guides/getting-started-with-alfresco-blueprint.md

```diff
@@ -1,151 +1,126 @@
-
-Alfresco was founded in 2005 by John Newton who, many years earlier, cofounded Documentum, one of the pioneers of document management. Alfresco Community is an enterprise-ready, Java-based platform used for document management, records management, image management and more. Features include versioning, Lucene-powered search, federated servers, clustering, roles-based permissions and more. You'll be in good company with Alfresco; other users include NASA, KLM, Merck and Ricoh.
-
+For more information, please visit [http://www.alfresco.com](http://www.alfresco.com)
 
 ### Audience
-
 CenturyLink Cloud Users
 
+### Impact
+After reading this article, the user should feel comfortable getting started using the Blueprint technology on CenturyLink Cloud.
 
-### Deploying Alfresco Community on a New Server
-
-Alfresco Community is available as a Blueprint for deployment on a **new server**.
-
-#### Steps
-
-
-1. **Locate the Blueprint in the Blueprint Library**
-
-  Starting from the CenturyLink Control Panel, navigate to the Blueprints Library. Search for **Alfresco Community on Linux** in the keyword search on the right side of the page.
-
-2. **Click the Deploy Blueprint button.**
-
-3. **Set Required parameters.**
-
-  Set the following parameters in addition to those associated with your server itself (password, network, group, etc.):
-
-  * **Service Password** -  Provide service password 6 chars or more 
-  * **Tomcat server shutdown port** -  Tomcat Shutdown Port default 8005
-  * **Apache server SSL port** - default 443
-  * **Libreoffice port** -  LibreOffice Server Port default 8100
-  * **Apache server port** - default 80
-  * **User's name** -  Users Name 
-  * **JDBC database** -  Database name default alfresco
-  * **Alfresco RMI port** -  Port default 50500
-  * **Alfresco_ftp_port** -  Port default 21
-
-5. **Review and Confirm the Blueprint**
-
-6. **Deploy the Blueprint**
-
-  Once verified, click on the `deploy blueprint` button. You will see the deployment details stating the Blueprint is queued for execution.
-
-  This will kick off the Blueprint deploy process and load a page where you can track the deployment progress. Deployment will typically complete within five minutes.
-
-7. **Deployment Complete**
-
-  Once the Blueprint has finished executing on your server you may access Alfresco Community by navigating to your server via http.
-
-8. **Enable public access** (optional)
-
-  Servers are built using private IPs only with access with client or IPSEC VPN.  For inbound access from the Internet add a public IP to your master server.
-
-  <a href="../../../network/how-to-add-public-ip-to-virtual-machine/">
-    <img style="border:0;width:50px;vertical-align:middle;" src="/knowledge-base/images/shared_assets/fw_icon.png">
-    Adding a public IP to your virtual machine
-  </a>
-
-
+### Prerequisite
+- Access to the CenturyLink Cloud platform as an authorized user.
```

Comment:
No It doesn't.

Bob Stolzberg | Sr. Cloud Integration Engineer
CenturyLink
Mobile/Text: (314) 397-2610
E-Mail: bob@CTL.io
www.CenturylinkCloud.com


From: Rathromony Ros <notifications@github.com<mailto:notifications@github.com>>
Reply-To: CenturyLinkCloud/PublicKB <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tue, 7 Jul 2015 10:37:32 -0700
To: CenturyLinkCloud/PublicKB <PublicKB@noreply.github.com<mailto:PublicKB@noreply.github.com>>
Cc: User <bob.stolzberg@centurylink.com<mailto:bob.stolzberg@centurylink.com>>
Subject: Re: [PublicKB] Ecosystem july (#205)


In Ecosystem Partners/Marketplace Guides/getting-started-with-alfresco-blueprint.md<https://github.com/CenturyLinkCloud/PublicKB/pull/205#discussion_r34067361>:

> -7. **Deployment Complete**
> -
> -  Once the Blueprint has finished executing on your server you may access Alfresco Community by navigating to your server via http.
> -
> -8. **Enable public access** (optional)
> -
> -  Servers are built using private IPs only with access with client or IPSEC VPN.  For inbound access from the Internet add a public IP to your master server.
> -
> -  <a href="../../../network/how-to-add-public-ip-to-virtual-machine/">
> -    <img style="border:0;width:50px;vertical-align:middle;" src="/knowledge-base/images/shared_assets/fw_icon.png">
> -    Adding a public IP to your virtual machine
> -  </a>
> -
> -
> +### Prerequisite
> +- Access to the CenturyLink Cloud platform as an authorized user.


Does this need to be a bulleted item?

-
Reply to this email directly or view it on GitHub<https://github.com/CenturyLinkCloud/PublicKB/pull/205/files#r34067361>.

This communication is the property of CenturyLink and may contain confidential or privileged information. Unauthorized use of this communication is strictly prohibited and may be unlawful. If you have received this communication in error, please immediately notify the sender by reply e-mail and destroy all copies of the communication and any attachments.